bolt/metrics.go

```diff
@@ -107,7 +116,24 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 		telegrafs = tx.Bucket(telegrafBucket).Stats().KeyN
 		tokens = tx.Bucket(authorizationBucket).Stats().KeyN
 		users = tx.Bucket(userBucket).Stats().KeyN
-		return nil
+		err := tx.Bucket(telegrafPluginsBucket).ForEach(func(k, v []byte) error {
```

Comment:
did you consider doing this stat counting incrementally as the telegraf configs come in? this setup won't extedn into cloud as is.... I'm pretty sure 🤔

Reply:
yes, but this seemed to be the simplest way of tracking plugins configured in every config on the system. I suppose we could add more logic for when a config is updated vs newly added. on update, we'd diff the old config and apply it to the global configured plugins list..

do you know who's consuming these stats and how often?

Reply:
heh, I was goign to ask you that exact question lol. To me its a service concern anyhow, less so a store concern... not a hill for me to die on though

Reply:
i'm completely open to other ideas regarding how to keep this aggregated list up to date. this seemed minimally invasive, though i have no hard backed estimates on how performant this would/not be on cloud.

Reply:
this wont' even run in cloud, this is bolt specific it looks like, OSS only

Reply:
cloud specific stuff would be implemented in idpe though?? or what am i missing. a discussion for next year perhaps

Reply:
@glinton I think that you'd want to implement the abstraction as a part of the `kv` package. That'd be the best place to get these kinds of metrics.